controllers/keystoneapi_controller.go

```diff
@@ -38,6 +38,7 @@ import (
 	util "github.com/openstack-k8s-operators/lib-common/modules/common/util"
 	database "github.com/openstack-k8s-operators/lib-common/modules/database"
 	mariadbv1 "github.com/openstack-k8s-operators/mariadb-operator/api/v1beta1"
+	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
This does not need to be named import. You can simply have
```
"sigs.k8s.io/controller-runtime/pkg/log"
```
and then later use
```
log.FromContext(...)
```

Reply:
ok